addons/website_sale_stock/static/tests/tours/website_sale_stock_combo_configurator.js

```diff
@@ -0,0 +1,49 @@
+import { registry } from '@web/core/registry';
+import configuratorTourUtils from '@sale/js/tours/combo_configurator_tour_utils';
+import stockConfiguratorTourUtils from '@website_sale_stock/js/tours/combo_configurator_tour_utils';
+
+registry
+    .category('web_tour.tours')
+    .add('website_sale_stock_combo_configurator', {
+        url: '/shop?search=Combo product',
+        steps: () => [
+            {
+                content: "Select Combo product",
+                trigger: '.oe_product_cart a:contains("Combo product")',
+                run: 'click',
+            },
+            {
+                content: "Click on add to cart",
+                trigger: '#add_to_cart',
+                run: 'click',
+            },
+            configuratorTourUtils.assertQuantity(1),
+            // Assert that it's impossible to add less than 1 product.
```

Comment:
you have a constraint in `sale`   module but check here https://github.com/odoo/odoo/pull/186645/files#diff-61d6ee91447846eb598fbae13bc79ae91f5a6497713c14eaa5cdc50c36e32218 or I messed it up?

Reply:
It is just that someone can remove the check in sale module and not think that it is expected behavior for sale module as well but it may seem unrealistic, so up to you :)